js/app.js

```diff
@@ -0,0 +1,148 @@
+'use strict';
+
+let resourceButton = document.getElementById('requestResourceButton');
+resourceButton.addEventListener('click', giveInformation);
+
+function giveInformation() {
+  let inputBox = document.getElementById('resourceId').value;
```

Comment:
Pick better variable names that describe the data that's actually in the variable. There could be multiple input boxes and I wouldn't know which one your talking about.